src/features/activity/AddressActivityItem.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { StyleSheet } from 'react-native'
+import { useTranslation } from 'react-i18next'
+import Box from '../../components/Box'
+import Text from '../../components/Text'
+import { useColors } from '../../theme/themeHooks'
+import TouchableOpacityBox, {
+  TouchableOpacityBoxProps,
+} from '../../components/TouchableOpacityBox'
+import AccountIcon from '../../components/AccountIcon'
+import { useAccountStorage } from '../../storage/AccountStorageProvider'
+import Menu from '../../assets/images/menu.svg'
+import { ellipsizeAddress } from '../../utils/accountUtils'
+
+export const LIST_ITEM_HEIGHT = 70
+export type AddressActivityItemProps = {
+  accountAddress: string
+  showBubbleArrow?: boolean
+  onMenuPress?: () => void
+} & TouchableOpacityBoxProps
```

Comment:
this should be `} & BoxProps<Theme>`

Reply:
Hmm doesn't `TouchableOpacityBox` have different props. Will try but I remember I was running into something